<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRecordDialog</class>
 <widget class="QDialog" name="QRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>99</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>99</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>99</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>1</y>
     <width>192</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_back">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../qfi.qrc">
        <normaloff>:/qfi/images/back.png</normaloff>:/qfi/images/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_play">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../qfi.qrc">
        <normaloff>:/qfi/images/pause.png</normaloff>:/qfi/images/pause.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ahead">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../qfi.qrc">
        <normaloff>:/qfi/images/ahead.png</normaloff>:/qfi/images/ahead.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qfi.qrc"/>
 </resources>
 <connections/>
</ui>
